app/models/concerns/child_minder.rb

```diff
@@ -30,6 +58,24 @@ def delete_children(page)
     end
   end
 
+  def restore_children(page)
+    if self.class.include? TrashableModel
+      paginated_children(page).each do |child|
+        child.current_transaction = current_transaction if child.class.include? ChildMinder
+        child.update(is_deleted: false)
+      end
+    end
+  end
+
+  def purge_children(page)
+    if self.class.include? TrashableModel
+      paginated_children(page).each do |child|
+        child.current_transaction = current_transaction if child.class.include? ChildMinder
```

Comment:
Seems odd to test for `ChildMinder` when you're setting `#current_transaction`. Would it make sense to test for `Transactionable` instead?